packages/core/test/test-util/mocked-access-tokens.ts

```diff
@@ -25,6 +26,16 @@ const subscriberServiceTokenPayload = {
 
 export const subscriberServiceToken = signedJwt(subscriberServiceTokenPayload);
 
+// This tokens are used to test cases when as a user Jwt only {iss:someXSUAAurl} is provided.
+// See docs on [[DestinationAccessorOptions]] for mor details
```

Comment:
```suggestion
// See docs on [[DestinationAccessorOptions]] for more details.
```

Reply:
This doc comment is intended for internal use only?

Reply:
yes